lib/Doctrine/Migrations/Events.php

```diff
@@ -4,6 +4,9 @@
 
 namespace Doctrine\Migrations;
 
+/**
+ * @internal
```

Comment:
this must not be internal. The events are extension points of the library, so the event names need to be public (otherwise they are not usable).

Reply:
Fixing.